src/app/collections/Collections.js

```diff
@@ -0,0 +1,159 @@
+import React, { Component } from "react";
+
+import config from "../../config";
+import * as api from "../apiActions";
+import * as analytics from "../../components/GoogleAnalytics";
+import Octicon from "../../components/Octicon";
+import ReviewNavigation from "../review/ReviewNavigation";
+import { setSavedDecks, getSavedDecks } from "../utils/savedDecks";
+import SkillProgress from "../decks/SkillProgress";
```

Comment:
Why Skill Progress instead of HabitTracker?

Reply:
Completely arbitrary but I like thinking of the collections pages as a skill (i.e. the JavaScript "skill"). I think people would be more interested in seeing their progress on this sub page.

Reply:
I like the concept, especially when user accounts are eventually added, so users can see which "skills" they've mastered etc. But maybe we later revist trying to fit the HabitTracker somewhere on the page.